File: Java/src/main/java/ch/scs/cc/testing/smells/FlightDto.java
package ch.scs.cc.testing.smells;

import org.inferred.freebuilder.FreeBuilder;

@FreeBuilder
public interface FlightDto {

    String originCity();
    String originAirportCode();
    String destinationCity();
    String destinationAirportCode();

    class Builder extends FlightDto_Builder {
    }
}

File: CSharp/BackingCode/FlightDto.cs
namespace BackingCode
{
    public record FlightDto
    {
        public string OriginCity { get; init; }
        public string OriginAirportCode { get; init; }
        public string DestinationCity { get; init; }
        public string DestinationAirportCode { get; init; }
    }
}
